src/main/java/sirius/biz/process/Processes.java

```diff
@@ -401,6 +401,7 @@ private boolean modify(String processId, Predicate<Process> checker, Consumer<Pr
                 return true;
             } catch (OptimisticLockException e) {
                 Wait.randomMillis(250, 500);
+                process = elastic.find(Process.class, processId).orElse(null);
```

Comment:
shouldnt we move the if process null return false into the while loop then to prevent NPEs for deleted processes (shouldnt be really neccessary)